taskcall/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd54da03-0a8c-4796-aaa6-61eeb04e611b",
+  "app_id": "taskcall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and centralize Datadog incidents with TaskCall",
+    "title": "TaskCall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Incident Notifications",
+        "image_url": "images/incident_notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Incident Details",
+        "image_url": "images/incident_details.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Impact Visibility and Status Dashboard",
+        "image_url": "images/impact_visibility_status_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "On-Call Management",
+        "image_url": "images/on_call_management.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Incidents",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
```suggestion
      "Supported OS::macOS",
      "Queried Data Type::Incidents"
```

Reply:
Jumping in here for Ana while she's out of office. Just adding one last tag to clarify that this pulls out incidents. This is the last change and then we can publish this to your sandbox.